registration.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>good.teens</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="registration.css">
+    <script src="reg2.js"></script>
```

Comment:
this is not the place to attach scripts and I don't think this script exist but we can fix that in future prs

Reply:
Does exist now and script attached correctly now. Thanks